runtime/functions

```diff
@@ -259,8 +259,11 @@ trust_localnet() {
 }
 
 set_resolvconf_perms() {
-  echo "Setting resolv ACLs..."
-  setfacl -m user:${PG_USER}:r /etc/resolv.conf
+  if setfacl -m user:${PG_USER}:r /etc/resolv.conf; then
+    echo "Setting resolv.conf ACLs..."  
+  else
+    echo "resolv.conf ACLs couldn't be set: `setfacl` not supported"
+  fi
 }
```

Comment:
I'd prefer 

```bash
echo "Setting resolv.conf ACLs..."
setfacl -m user:${PG_USER}:r /etc/resolv.conf || true
```

I cannot understand why you would see this error inside the container.

> ```
> $ setfacl || true
> $ setfacl: not found
> ```